.github/ISSUE_TEMPLATE/4_KIT_release_checks.md

```diff
@@ -5,23 +5,31 @@ title: "[KIT NAME] Release Checks"
 labels: kit
 ---
 
-This issue tracks all relevant release checks for KITs.
+This issue tracks all relevant release checks for your KIT.
+
+> [!IMPORTANT]  
+> Follow the [guidance on how to use the templates](https://github.com/eclipse-tractusx/sig-release/blob/main/README.md#release-management-acceptance-criteria).
 
 ## Release Info
 
 Please provide information on what you want to be included in the Eclipse Tractus-X release.
 If you are not owner of this issue, please provide the information as comment to the issue.
+Make sure to assign this issue to expert(s) for their approval, as soon as you have finished preparation.
+Best practice: have a professional Peer review your prepared content before you submit for expert approvals.
+
+**Version to be included in Eclipse Tractus-X release:** *version placeholder*
 
 If changes to your KIT are not yet live on [our website](https://eclipse-tractusx.github.io/developer), link the open PR introducing the changes for this release.
 
 If changes are already live, also point out the PRs witch included the changes for this version if possible.
-Please refer to TRG 8.03 for the classification of mandatory and optional artefacts.
 
 - [PR number](pr-url)
 
 # General
 
 - [ ] **CHANGELOG** available and maintained
+- [ ] [**Tractus-X Release Guidelines**](https://eclipse-tractusx.github.io/docs/release) fulfilled (relevant KIT TRGs tba)
```

Comment:
KIT TRGs will be published next week. We can add the link later.

Reply:
IMHO the text and the existing link are ok and sufficient here - IF the KIT TRGs next week will end up in the same area (that is my assumption).
Proposal: delete the text in (brackets)
=> therefore the template is ready to go